core/src/test/java/nl/nn/adapterframework/util/CounterStatisticTest.java

```diff
@@ -0,0 +1,64 @@
+package nl.nn.adapterframework.util;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+
+/**
+ * CounterStatistic Tester.
+ *
+ * @author <Sina Sen>
+
+ */
+public class CounterStatisticTest {
+
+
+    /**
+     *
+     * Method: performAction(int action)
+     *
+     */
+    @Test
+    public void testPerformAction01() throws Exception {
+        CounterStatistic cs = new CounterStatistic(10);
+        cs.performAction(0);
+        assertEquals(cs.getValue(), 10);
+    }
+
+    /**
+     *
+     * Method: performAction(int action)
+     *
+     */
+    @Test
+    public void testPerformAction2() throws Exception {
+        CounterStatistic cs = new CounterStatistic(10);
+        cs.performAction(2);
+        assertEquals(cs.getValue(), 0);    }
+
+    /**
+     *
+     * Method: performAction(int action)
+     *
+     */
+    @Test
+    public void testPerformAction34() throws Exception {
+        CounterStatistic cs = new CounterStatistic(10);
+        cs.performAction(3);
+        assertEquals(cs.getValue(), 10);    }
```

Comment:
code style dictates that the closing bracket, when present should be on a new line.

Applies in multiple tests

Reply:
changed it